eogrow/pipelines/byoc.py

```diff
@@ -101,7 +101,7 @@ def _get_tiff_paths(self) -> List[str]:
     def _get_byoc_compliant_path(self, relative_path: str) -> str:
         """Transforms a project relative path to a bucket-name relative path that is required by BYOC."""
         absolute_path = fs.path.combine(self.config.storage.project_folder, relative_path)  # type: ignore[attr-defined]
-        return absolute_path[6 + len(self.bucket_name) :]  # removes s3://<bucket-name>/
+        return absolute_path.replace(f"s3://{self.bucket_name}", "")  # removes s3://<bucket-name>/
```

Comment:
Perhaps unclear comment, since this doesn't remove the trailing fwd slash?

Reply:
great catch <3 The comment is correct, my change is not. It should remove the slash as well but I can't count for shit.